ingestor/chalicelib/ferry/s3.py

```diff
@@ -0,0 +1,48 @@
+import glob
+import boto3
+from io import BytesIO
+import gzip
+import os
+
+from disk import DATA_DIR
+
+
+s3 = boto3.client("s3")
+
+S3_BUCKET = "tm-mbta-performance"
+
+S3_DATA_TEMPLATE = "Events-live/{relative_path}.gz"
```

Comment:
We don't want this to be in the gobble dir, we can consider these the historic imports